docs/user/exercises/modeling.rst

```diff
@@ -1,2 +1,208 @@
+
+.. note::
+    Artemis uses semi-automatic grading of modeling exercises using Machine Learning.
+    You can hence train the model by selecting *Use in Assessment Training* checkbox while creating an Example Submission.
+
+    .. figure:: modeling/use-in-assessment-training.png
+              :align: center
+
+- Alternatively, you can also import modeling exercise from the existing one by clicking on **Import Modeling Exercise**
+
+    .. figure:: modeling/import-modeling-exercise.png
+              :align: center
+
+
+  Result: **Modeling Exercise**
+
+    .. figure:: modeling/course-dashboard-exercise-modeling.png
+              :align: center
+
+- Click the |edit| button of the modeling exercise and adapt the interactive problem statement. There you can also set release and due dates.
+- Click the |scores| button to see the scores achieved by the students.
+- Click the |participation| button to see the list of students participated in the exercise.
+- Click the |submission| button to see the list of submission submitted by students.
+- Click the |example-submission| button to modify/add example submission of the exercise.
+- Click the |delete| button to delete the exercise.
+- You can get an overview of the exercise by clicking on the title.
+
+Student Submission
+------------------
+
+- When the exercise is released students can work on the exercise.
+
+    .. figure:: modeling/modeling-exercise-card-student-view.png
+              :align: center
+
+- They can start the exercise by clicking the |start| button.
+
+- Once they start the exercise, they will now have the option to work on it in an online modeling editor by clicking on  the |open-modeling-editor| button.
+
+- The screenshot below depicts the online modeling exercise interface for students. They can read the Problem Statement, work on the online editor and also provide an explanation to their solutions, if needed.
+
+    .. figure:: modeling/modeling-exercise-students-interface.png
+              :align: center
+
+Assessment
+----------
+
+When the due date is over you can assess the submissions.
+
+- To assess the submissions, first click on Assessment Dashboard
+
+    .. figure:: modeling/assessment-dashboard.png
+              :align: center
+
+- Then click on Submissions of the modeling exercise
+
+    .. figure:: modeling/exercise-dashboard.png
+              :align: center
+
+- You will then be redirected to *Submissions* Dashboard
+
+    .. figure:: modeling/submissions-dashboard.png
+              :align: center
+
+- Click on |assess-submission| button of specific student. You will then be redirected to the assessment page where you will be able to assess submission of that student.
+
+- You can now start assessing the elements of the model by double clicking it. Once you double click, you will get an assessment modal where you can assign points, feedback and navigate through all other assessable components.
```

Comment:
```suggestion
- You can now start assessing the elements of the model by double clicking it. Once you double click, you will get an assessment dialog where you can assign points, feedback and navigate through all other assessable components.
```
I think being more generic in modal vs. dialogue is better here for the end user documentation.